includes/admin/pages.php

```diff
@@ -225,7 +225,9 @@ public function html()
     		if ($tab_id === $current_tab) {
 
     			echo '<div class="simcal-bg-white simcal-p-[2%] simcal-w-[54%] simcal-rounded-[5px] ">';
-
+    			if ($current_tab == 'feeds') {
+    				do_action('simcal_add_auth_via_google_button');
```

Comment:
Name could be `simple_calendar_auth_via_google_button` and `simple_calendar_auth_via_xtendify_button`

Reply:
Done